src/components/LabwareFinder.vue

```diff
@@ -58,6 +60,8 @@ export default {
           this.showAlert(res.errors, 'danger')
         }
       })
+      // redirecting the focus to the input after searching
+      this.$refs.search.$refs.inputRef.focus()
```

Comment:
Shouldn't be this.$refs.inputRef.focus() ?

Reply:
I think I need to reference _search_ first (finds the traction-input component that LabwareFinder uses) to be able to reference _inputRef_ which finds the LabwareFinder